sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MethodsShouldUseBaseTypes.cs

```diff
@@ -179,10 +179,10 @@ private ITypeSymbol FindParameterUseAsType(IdentifierNameSyntax identifier, Sema
             {
                 return semanticModel.GetTypeInfo(identifier).ConvertedType;
             }
-            else if (identifierParent is ElementAccessExpressionSyntax)
-            {
+            else if (identifierParent is ElementAccessExpressionSyntax)
```

Comment:
Same question you asked. What changed here?

Reply:
As we discussed, let's leave it for now (in both PRs) and fix properly - perhaps you'll be more successful with gitattributes...